CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 8.1.4
```

Comment:
Should be added as `HEAD`. Versioning concerns are not PR concerns.

Reply:
Can you elaborate? What should I change it to?

Reply:
It should literally just be `HEAD`, as previous PRs have done it as well. Assigning a real version to this is part of the release procedure.

Reply:
No you did not get it. This is not part of 8.1.3, which is already released. The convention is to literally write "HEAD" as the version.

Reply:
Done.